Doc/howto/clinic.rst

```diff
@@ -1290,9 +1289,9 @@ after ``self``, or, if ``self`` is not used, as the first argument.  The argumen
 will be of type ``PyTypeObject *``.  The argument will not appear in the
 :py:attr:`!__text_signature__`.
 
-The ``defining_class`` converter is not compatible with
-:py:meth:`!__init__` and :py:meth:`!__new__`
-methods, which cannot use the :data:`!METH_METHOD` convention.
+The ``defining_class`` converter is not compatible with :py:meth:`!__init__`
+and :py:meth:`!__new__` methods, which cannot use the :ref:`METH_METHOD
```

Comment:
BTW you can now use ``` :c:macro:`METH_METHOD` ``` if you prefer.

Reply:
Oh, that's nice; thanks! I'll apply that before landing.